pkg/virt-launcher/virtwrap/cli/libvirt.go

```diff
@@ -167,7 +167,7 @@ func (l *LibvirtConnection) DomainEventDeviceRemovedRegister(callback libvirt.Do
 	}
 
 	l.domainDeviceRemovedEventCallbacks = append(l.domainDeviceRemovedEventCallbacks, callback)
-	_, err = l.VolatileDomainEventDeviceRemovedRegister(callback)
+	_, err = l.VolatileDomainEventDeviceRemovedRegister(nil, callback)
```

Comment:
Having a `nil` arguments reduce readability can we have a pair of functions for this ? 

```golang
VolatileDomainEventDeviceRemovedRegister(callback libvirt.DomainEventDeviceRemovedCallback) (int, error)
VolatileDomainEventDeviceRemovedRegisterWithDomain(domain VirDomain, callback libvirt.DomainEventDeviceRemovedCallback) (int, error)
```

Reply:
This is exposing the original API as well.
In this specific case it allows also to use the same function through the original usage, making it functionally covered even if no SR-IOV scenario is in place. If there is a problem with it, we will immediately know about it.